clang/test/lit.site.cfg.py.in

```diff
@@ -22,7 +22,7 @@ config.host_cxx = "@CMAKE_CXX_COMPILER@"
 config.llvm_use_sanitizer = "@LLVM_USE_SANITIZER@"
 config.have_zlib = @LLVM_ENABLE_ZLIB@
 config.have_zstd = @LLVM_ENABLE_ZSTD@
-config.clang_arcmt = @CLANG_ENABLE_ARCMT@
+config.clang_objc_rewriter = "@CLANG_ENABLE_OBJC_REWRITER@" == "ON"
```

Comment:
Instead of this, please add `CLANG_ENABLE_OBJC_REWRITER` to the `llvm_canonicalize_cmake_booleans` list at the top of clang/test/CMakeLists.txt and do `clang_objc_rewriter = @CLANG_ENABLE_OBJC_REWRITER@` here.

Reply:
I was wondering what the right way of doing this was; I’ll make a new pr for that

Reply:
Opened #125117.